index.d.ts

```diff
@@ -49,16 +49,16 @@ type ZoneLetter = 'X'
 
 declare class UTMLatLng {
 
-Constructor(type?: ElipsoidType)
+constructor(type?: ElipsoidType)
 
-ConvertLatLngToUtm(latitude: number, longitude: number, precision: number): string | {
```

Comment:
Changed "ConvertLatLngToUtm" to "convertLatLngToUtm" to follow method naming conventions in TypeScript, where method names should start with a lowercase letter.